### ISSUE TITLE
Confusing message: The installed Angular CLI version is older than the latest stable version.

### ISSUE THREAD

**Issue by johnpapa** [angular/angular-cli#16885]
# 🐞 bug report

I first installed v9.0.1 of the @angular/cli globally. Then I ran  ...

`ng update @angular/cli @angular/core`

It complains about this ...

```
The installed Angular CLI version is older than the latest stable version.
Installing a temporary version to perform the update.
```

If it means the global CLI, then how is this possible if I have 9.0.1 and I just installed it moments ago?

This doesnt make sense to me ... either the message is wrong, or I am misunderstanding it.

**Comment by alan-agius4** [comment]
@johnpapa, the check their is for the local CLI.

Maybe changing the message to `The installed local Angular CLI version is older than the latest stable version.` should ease the confusion?

**Comment by johnpapa** [comment]
yes, that would be MUCH clearer. thanks for clarifying what is happening.

I added a quick PR https://github.com/angular/angular-cli/pull/16888

**Comment by clydin** [comment]
Fixed via #16892
